kafka/pom.xml

```diff
@@ -198,7 +198,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.11</artifactId>
-      <version>0.9.0.0</version>
```

Comment:
0.9.0.1 fixed some bugs, it should be more stable than 0.9.0.0. I think it's better to upgrade before a lot of customers using it. Following is Kafka 0.9.0.1 release notes: http://apache.claz.org/kafka/0.9.0.1/RELEASE_NOTES.html